CHANGELOG.md

```diff
@@ -23,6 +23,8 @@
 * [FEATURE] Query Frontend: Add `cortex_query_fetched_series_total` and `cortex_query_fetched_chunks_bytes_total` per-user counters to expose the number of series and bytes fetched as part of queries. These metrics can be enabled with the `-frontend.query-stats-enabled` flag (or its respective YAML config option `query_stats_enabled`). #4343
 * [FEATURE] AlertManager: Add support for SNS Receiver. #4382
 * [FEATURE] Distributor: Add label `status` to metric `cortex_distributor_ingester_append_failures_total` #4442
+* [ENHANCEMENT] Querier: Add setting in query_range middleware to configure the set of headers from the requests to be forwared to downstream requests.The following setting is added. #4486
```

Comment:
The people reading this are Cortex admins, not developers.  Write it in terms of services, not specific code structs.
Also spelling: "forwared".